Source/Core/Core/HW/EXI_DeviceSlippi.cpp

```diff
@@ -1980,7 +1981,7 @@ void CEXISlippi::prepareOnlineMatchState()
 
 #ifdef LOCAL_TESTING
 	localPlayerIndex = 0;
-	chatMessageId = localChatMessageId;
+    sentChatMessageId = localChatMessageId;
```

Comment:
Can you run the auto-formatter? There are a lot of formatting issues in this.

Reply:
Sure.
I can never get it to match the codebase even if I select clang, let me fix that!

Reply:
Done @JLaferri